cg/services/order_validation_service/workflows/balsamic_umi/constants.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+from cg.constants import DataDelivery
+
+
+class BalsamicUmiDeliveryType(Enum):
```

Comment:
Balsamic-UMI seems to have support for more delivery options such as Scout as well

![image](https://github.com/user-attachments/assets/84c638c5-99b3-483e-9a26-ce83a890cb7b)

Reply:
nice catch